packages/ordered-list/src/ListItem/ListItem.styles.ts

```diff
@@ -0,0 +1,51 @@
+import { css } from '@leafygreen-ui/emotion';
+import { createUniqueClassName, Theme } from '@leafygreen-ui/lib';
+import { color, spacing, transitionDuration } from '@leafygreen-ui/tokens';
+
+export const contentClassName = createUniqueClassName('content');
+export const stepIconClassName = createUniqueClassName('step');
+
+const STEP_SIZE = 20;
```

Comment:
Took me a while to figure out what step means. Is there a better name for this?

Reply:
Updated